tests/holodeck.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: HOLODECK_NAME
+  description: "end-to-end test infrastructure"
+spec:
+  provider: aws
+  auth:
+    keyName: cdesiniotis
```

Comment:
```suggestion
    keyName: cnt-ci
```

Reply:
Updated.

Reply:
@ArangoGutierrez can you add the AWS_SSH_KEY secret to this repo associated with cnt-ci?